src/styles/index.scss

```diff
@@ -1,3 +1,213 @@
-body {
-  margin: 0;
+$color-bg-white: #fff;
+$color-text-dark: #34568b;
+$color-text-gray: #616070;
+$main-color: #00acdc;
+
+.page {
+  font-family: Roboto, Arial, Helvetica, sans-serif;
+
+  &__body {
+    margin: 0;
+  }
+}
+
+header {
+  background-color: $color-bg-white;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+}
+
+.logo {
+  display: block;
+  height: 40px;
+  width: 40px;
+}
+
+.nav {
+  &__list {
+    list-style: none;
+    display: flex;
+    text-transform: uppercase;
+    justify-content: center;
+    padding: 0;
+    margin: 0;
+  }
+
+  &__item + &__item {
+    margin-left: 20px;
+  }
+
+  &__link {
+    position: relative;
+    display: flex;
+    align-items: center;
+    font-family: Roboto, sans-serif;
+    text-decoration: none;
+    color: #060b35;
+    font-size: 12px;
+    font-weight: 500;
+    text-align: center;
+    height: 60px;
+    line-height: 40px;
+
+    &:hover {
+      color: $main-color;
+    }
+  }
+
+  .is-active {
+    color: $main-color;
+    position: relative;
+
+    &::after {
+      content: '';
+      position: absolute;
+      bottom: 0;
+      left: 0;
+      right: 0;
+      height: 4px;
+      border-radius: 8px;
+      background-color: $main-color;
+    }
+  }
+}
+
+// ****************** Cards ************************
+
+.cards {
+  display: grid;
+  grid-template-columns: repeat(200px);
```

Comment:
The 'grid-template-columns' property is incorrectly set to 'repeat(200px)'. It should specify the number of columns and their size, for example, 'repeat(1, 200px)' for a single column layout.